### DISCUSSION TITLE
[ISR] Persist  page cache between deployments

### DISCUSSION THREAD

**Discussion by nicosh** [vercel/next.js#40673] [Help]
Hi, 
As the title says is possible to persist page cache between deployments? 
My pages are not using any revalidate time, i will revalidate each page only when needed with `revalidate()`, the  problem is that after each deployment  the pages will be rebuilt after the first visit occurs, while would be nice if i can re-use the previous cache and revalidate them only if needed with on demand revaldiation.
Is this possible?

**Comment by victorcarvalhosp** [comment]
Do you have some update on this? Although they say ISR should cache between deployments, this is not the behavior we're getting on our end. We're not using any revalidate time as well, and using the pages router.
